version.js

```diff
@@ -151,16 +151,30 @@ function tryParseVersionBranch(branchName) {
 
 /**
  * Checks the PR to see if the major version should be incremented based on labels.
+ * @param {{ owner: string, repo: string }} repo
  * @param {string} commitMessage
- * @returns {Promise<'minor' | 'major'>}
+ * @returns {Promise<SemVer>}
  */
-async function getIncrementType(commitMessage) {
+async function getDefaultBranchNextVersion(repo, commitMessage) {
+  const previousRelease = await getLatestReleaseVersion(repo);
   const prNumber = tryParsePrNumber(commitMessage);
   if (prNumber === undefined) {
-    return "minor";
+    return previousRelease.inc("minor");
+  }
+  const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+  const pr = await octokit.rest.pulls.get({
+    ...repo,
+    pull_number: prNumber,
```

Comment:
Doesn't this assume that `prNumber` is exactly the title of the pull request we're looking for?

Reply:
This assumes the use of GitHub's default merge commit message which for squash and merge has the suffix of the PR reference in brackets e.g. `(#123)`

- E.g. https://github.com/pulumi/pulumi-azure/commit/8cb445e85cf630467346e4040bfbbaedff05c337

Reply:
`prNumber` is parsed from the `(#123)` suffix of the GH commit message - so we're looking up the PR by the number `123` - it's identifier.